index.d.ts

```diff
@@ -0,0 +1,4 @@
+// TypeScript Version: 3.5.2
+
+export declare function createWriter<T>(redis: T, key: string): T;
+export declare function createReader<T>(redis: T, key: string, opt?: T): T;
```

Comment:
You could optionally add types for `opt`. Its default values are defined like this:

```js
{
	live: true,
	waitTimeout: Infinity, // integer
	history: false,
	limit: Infinity // integer
}
```